libs/damap/src/lib/shared/search-field/search-field.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { MatFormFieldAppearance } from '@angular/material/form-field';
+import { UntypedFormControl } from '@angular/forms';
+
+@Component({
+  selector: 'damap-search-field',
+  templateUrl: './search-field.component.html',
+  styleUrl: './search-field.component.css',
+})
+export class SearchFieldComponent {
+  @Input() label: string = '';
+  @Input() placeholder: string = '';
+  @Input() appearance: MatFormFieldAppearance = 'outline';
+  @Input() control: UntypedFormControl;
```

Comment:
Trueee, that was a bad oversight 😅 
I had my console on element view the whole time and didnt notice the errors.
I will provide a default, since most search fields dont have a control connected to it.